`cargo install` fails with unresolve import of `rand_core::OsRng`

`cargo install --locked` works, so it's likely that `rand_core` did a patch version upgrade that broke its API? Note that `cargo install` without `--locked` won't use the `Cargo.lock` file.

```
$ cd innernet
$ cargo install --path client

error[E0432]: unresolved import `rand_core::OsRng`
  --> wireguard-control/src/key.rs:35:25
   |
35 |         use rand_core::{OsRng, RngCore};
   |                         ^^^^^ no `OsRng` in the root

error[E0432]: unresolved import `rand_core::OsRng`
  --> wireguard-control/src/key.rs:50:25
   |
50 |         use rand_core::{OsRng, RngCore};
   |                         ^^^^^ no `OsRng` in the root

warning: unused import: `RngCore`
  --> wireguard-control/src/key.rs:50:32
   |
50 |         use rand_core::{OsRng, RngCore};
   |                                ^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

warning: unused import: `RngCore`
  --> wireguard-control/src/key.rs:35:32
   |
35 |         use rand_core::{OsRng, RngCore};
   |
```

--- a/wireguard-control/Cargo.toml
+++ b/wireguard-control/Cargo.toml
@@ -14,7 +14,7 @@ base64 = "0.13"
 hex = "0.4"
 libc = "0.2"
 log = "0.4"
-rand_core = "0.6"
+rand_core = { version = "0.6", features = ["getrandom"] }
 curve25519-dalek = "4.0.0-pre.2"
 
 [target.'cfg(target_os = "linux")'.dependencies]

